pages/How-to/guide/assigning-issue.md

```diff
@@ -1,4 +1,60 @@
-<div align="center">
-  <img src="https://user-images.githubusercontent.com/74750414/169188784-680014eb-ae5a-4062-b9c0-40276f28a261.png" />
+
+
+
+<div>
+  <h1>Creating and assigning an issue from a repository</h1>
+  
+  <p> Issues are important as they alert a maintainer to review and make new changes or receive information. These Information could include bugs, new ideas, feedback etc.
+
+They are also a number of ways to creating an issue from a GitHub repository. 
+    
+However, it could also use the following steps.</p>
+  
+</div>
+
+<div>
+
+<h5>  Step 1 </h5>
+
+<p> Navigate to the main repository and just below the repository name, click on issues and create new issues. </p>
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/rn6dj1b7hch7jxwu4tx0.jpg" />
 </div>
 
+<div>
+  <h5>  Step 2 </h5>
+
+
+
+<p> When the new issue page opens, you can write a title describing the issue. You could also attach files to show these issues clearly.</p>
+
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/i3ij94bbs3oij9yva18s.jpg" />
+
+<p><b> Assigning </b> someone to work on the identified issue can be done using the assignees button on the left panel. </p>
+
+
+  <img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/9yv4tzmhe8mby5y7w0a6.jpg" />
+
+<p> You can have up to 10 people assigned on an issue. </p>
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/0dc7q2ezxv00epiekiu5.jpg" />
+
+<p> You can also add a label tag, showing the type of issue for better explanation. </p>
+
+<p> <b> Open source secret </b> for open source newbies, searching for `good first issue`can bring up easy to fix issues. </p>
+
+<p> Thus, the tag system helps to make this categorization easier for others to find and fix.
+
+After Assigning and labeling, you can now submit.</p>
+
+<h3>Step 3</h3>
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/wvoeblg92h2kgzyhay42.jpg" />
+
+<p> Your issue becomes open after submitting and the issue count increases. The assigned party is also notified.
+
+And, that's one way to create and assign an issue from the GitHub page. </p>
+
+<img src= "https://dev-to-uploads.s3.amazonaws.com/uploads/articles/ozjqbngoagopk1s2slxu.gif" />
+Until later 
```

Comment:
I think the steps you have written for the process don't have the required spacing, the steps are getting camouflaged with the screenshots.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable